File: Examino/App/src/models/Raports/RaportViewModel.ts
export interface RaportViewModel {
  patientName: string;
  patientSurname: string;
  doctorName: string;
  doctorSurname: string;
  raport: {
    id: string;
    patientId: string;
    doctorId: string;
    raportTime: Date;
    symptoms: string;
    examination: string;
    diagnosis: string;
    recommendation: string;
    comment: string;
  };
  prescription: {
    id: string;
    raportId: string;
    medicines: string;
  };
}

File: Examino/Server/Examino.Application/Functions/Raports/Queries/GetUserRaports/RaportViewModel.cs
using System;

namespace Examino.Application.Functions.Raports.Queries.GetUserRaports
{
    public record RaportViewModel
    {
        public Guid Id { get; init; }
        public string PatientName { get; init; }
        public string PatientSurname { get; init; }
        public string DoctorName { get; init; }
        public string DoctorSurname { get; init; }
        public DateTimeOffset RaportTime { get; init; }
        public string Symptoms { get; init; }
        public string Examination { get; init; }
        public string Diagnosis { get; init; }
        public string Recommendation { get; init; }
        public string Comment { get; init; }
        public Guid PrescritpionId { get; init; }
        public string Medicines { get; init; }
    }
}
